src/common/Header/styled.js

```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+import { ReactComponent as VideoIcon } from "./VideoIcon.svg";
+import { ReactComponent as SearchIcon } from "./SearchIcon.svg";
+
+export const Wrapper = styled.header`
+    background: ${({ theme }) => theme.color.woodsmoke};
+    height: 94px;
+    display: grid;
+    gap: 80px;
+    grid-template-columns: 1.5fr 1fr 2fr;
+    padding: 35px 16px 35px 16px;
+    align-content: center;
+    color: ${({ theme }) => theme.color.white};
+    width: 100%;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        height: 142px;
+        grid-template-columns: 1fr 1fr;
+        padding: 16px;
+        gap: 19px;
+    }
+`;
+
+export const TitleWrapper = styled.div`
+    display: flex;
+    gap: 12px;
+    align-items: center;
+    justify-self: right;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        gap: 8px;
+        margin-top: 8px;
+        justify-self: left;
+    }
+`;
+
+export const StyledVideoIcon = styled(VideoIcon)`
```

Comment:
Let's give it `flex-shrink: 0;`

![image](https://user-images.githubusercontent.com/117176251/220739303-dc134faa-feb3-4c17-a0b3-95f4d58bcd7d.png)